pkg/cmd/server/api/types.go

```diff
@@ -40,6 +49,10 @@ type NodeConfig struct {
 	// PodManifestConfig holds the configuration for enabling the Kubelet to
 	// create pods based from a manifest file(s) placed locally on the node
 	PodManifestConfig *PodManifestConfig
+
+	// DockerExecHandlerName is the name of the handler to use for executing
+	// commands in Docker containers.
+	DockerExecHandlerName DockerExecHandlerType
```

Comment:
A stanza for docker options would probably be good. If we'd known ahead of time, I'd have put `AllowDisabledDocker` in there as well